### ISSUE TITLE
Various incorrect spaces are not fixed

### ISSUE THREAD

**Issue by mesqueeb** [TheRealSyler/sass-formatter#30]
**Describe the bug**
Various double spaces are not fixed

**To Reproduce**
```sass
@import  'something'
.animate-blink
  animation : blink  1.3s infinite
@keyframes  blink
  from
    opacity : 1. 0
```

list of spaces I'd expect to be fixed:
- extra spaces after `@import`
- extra spaces before `:`
- extra spaces in between `blink   1.3s infinite`
- extra spaces after `@keyframes`
- extra spaces in between `1. 0`

**Expected behavior**
to be fixed to:
```sass
@import 'something'
.animate-blink
  animation: blink 1.3s infinite
@keyframes blink
  from
    opacity: 1.0
```

**Desktop (please complete the following information):**
macOS, latest VSCode with this extension

**Comment by TheRealSyler** [comment]
i'm aware, i will fix it when i have time and want to do it.

**Comment by mesqueeb** [comment]
@TheRealSyler this is such an awesome fix! Happy to see you're still maintaining! : )

I hope Vetur adopts your latest version soon as well! ; )

**Comment by mesqueeb** [comment]
@TheRealSyler 
two remaining issues
important:
```
opacity: 1. 0
// not fixed to
opacity: 1. 0
```

```
opacity: 1. 0
// not fixed to
opacity: 1. 0
```

**Comment by mesqueeb** [comment]
@TheRealSyler 
two remaining issues
- important:
```
opacity : 1.0
// not fixed to
opacity: 1.0
```
- less important:
```
opacity: 1. 0
// not fixed to
opacity: 1.0
```

Shall I open a new issue for this?

**Comment by TheRealSyler** [comment]
@mesqueeb no, i will reopen this issue.

**Comment by TheRealSyler** [comment]
fixed but i want to fix #32 and #38 before i update the extension and the formatter.

**Comment by mesqueeb** [comment]
thanks, this was fixed!
